packages/api/cms-api/src/user-permissions/auth/user-permissions.guard.ts

```diff
@@ -50,7 +50,9 @@ export class UserPermissionsGuard implements CanActivate {
         if (requiredPermissions.length === 0) throw new Error(`RequiredPermission decorator has empty permissions in ${location}`);
         if (this.isResolvingGraphQLField(context) || skipScopeCheck) {
             // At least one permission is required
-            return requiredPermissions.some((permission) => this.accessControlService.isAllowed(user, permission));
+            return requiredPermissions
+                .filter((permission): permission is Permission => permission !== DisablePermissionCheck)
```

Comment:
I don't think this filter is needed, DisablePermissionCheck is already excluded above and the before this PR it was also not done.

(and it's basically an inline version of isPermission that still accepts anything (except DisablePermissionCheck) as Permission)

Reply:
We probably should use an as cast here instead, since we're adding an unnecessary runtime check for type safety.

Reply:
solved with 2f6bf750be05ba11c56deffcb10f7d1ea07a9c5c